### PR TITLE
Fix #118: Change version to 1.6.0-SNAPSHOT

### LINKED ISSUE #118: Change version to 1.6.0-SNAPSHOT
The version `1.5.2` is already released, new development needs to be done on snapshot version.

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: audit-base/pom.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
         <groupId>io.getlime.core</groupId>
         <artifactId>lime-java-core-parent</artifactId>
-        <version>1.5.2</version>
+        <version>1.6.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
     <artifactId>audit-base</artifactId>
-    <version>1.5.2</version>
+    <version>1.6.0-SNAPSHOT</version>
```
Comment (banterCZ) [inline audit-base/pom.xml:14]: I would rather omit the version completely. It is taken from parent.

File: audit-base/pom.xml [hunk 2]
```diff
@@ -6,12 +6,12 @@
     <parent>
         <groupId>io.getlime.core</groupId>
         <artifactId>lime-java-core-parent</artifactId>
-        <version>1.5.2</version>
+        <version>1.6.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
```
Comment (banterCZ) [inline audit-base/pom.xml:10]: May we remove this default value?

**Review by banterCZ — APPROVED** [review]

File: audit-base/pom.xml (hunk 1 above)
Comment (romanstrobl) [reply audit-base/pom.xml:14]: Ok.

File: rest-client-base/pom.xml [hunk 3]
```diff
@@ -49,7 +47,7 @@
         <dependency>
             <groupId>io.getlime.core</groupId>
             <artifactId>rest-model-base</artifactId>
-            <version>1.5.2</version>
+            <version>1.6.0-SNAPSHOT</version>
```
Comment (banterCZ) [inline rest-client-base/pom.xml:50]: Just one little thing, here we may use 
```suggestion
            <version>${project.version}</version>
```

**Review by banterCZ — APPROVED** [review]
Nice cleanup.

**Pull request merged**